addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SpringActuatorScanRule.java

```diff
@@ -196,15 +199,21 @@ private HttpMessage sendActuatorRequest(String encodingType, String actuatorEndp
         return null;
     }
 
-    private void raiseAlert(HttpMessage msg, int confidence, int risk) {
-        newAlert()
+    private AlertBuilder createAlert(HttpMessage msg, int confidence, int risk) {
```

Comment:
It seems like risk and confidence are always the same, so they could just be set here and not passed around.